content/en/docs/languages/go/generated-code-old.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Generated-code(non-generic) reference    
```

Comment:
Please put spaces before opening parentheses in text.

Maybe call this "legacy" instead?

Reply:
Done